app/src/main/cpp/Controller.h

```diff
@@ -42,6 +42,7 @@ struct Controller {
   uint32_t numAxes;
   bool leftHanded;
   bool inDeadZone;
+  vrb::TransformPtr pointerTransform;
```

Comment:
You forgot to add this to the Controller::operator=() function which is why it is broken on both Daydream and Wave. I'm not sure why it works on Go.